### ISSUE TITLE
Web Accessibility

### ISSUE THREAD

**Issue by GLC-Delivers** [jsor/lity#188]
I love this but it creates a possible accessibility trap for keyboard-only users. You cannot tell when the close button is in focus and the Esc key doesn't work when tabbing through content.

**Comment by NatalieMac** [comment]
Confirmed that this is the case. Having trouble currently with both YouTube and Vimeo videos. If you tab into the video controls, it's a keyboard trap and there's no way to exit from the video and close the lightbox without using the mouse. Breaks [WCAG 2.1.2 Keyboard Trap](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html).
